### PR TITLE
CLOUDP-185094: migration to SDK v1.0.0

### PR DESCRIPTION
**Pull request by wtrocki**

### CODE REVIEW

File: internal/cli/atlas/users/invite.go [hunk 1]
```diff
@@ -94,12 +94,12 @@ func (opts *InviteOpts) Run() error {
 
 const keyParts = 2
 
-func (opts *InviteOpts) createRoles() ([]atlasv2.RoleAssignment, error) {
+func (opts *InviteOpts) createRoles() ([]atlasv2.CloudRoleAssignment, error) {
 	if !config.IsCloud() {
 		return nil, nil
 	}
 
-	atlasRoles := make([]atlasv2.RoleAssignment, len(opts.orgRoles)+len(opts.projectRoles))
```
Comment (wtrocki) [inline internal/cli/atlas/users/invite.go:102]: Looks like we have two objects RoleAssignment and now CloudRoleAssignment (they were overriding each other).

File: internal/cli/atlas/users/invite.go [hunk 2]
```diff
@@ -147,27 +147,28 @@ func splitRole(role string) ([]string, error) {
...
-		return atlasv2.RoleAssignment{}, err
+		return atlasv2.CloudRoleAssignment{}, err
 	}
-	atlasRole := atlasv2.RoleAssignment{
-		GroupId: &value[0],
-		Role:    pointer.Get(strings.ToUpper(value[1])),
+	atlasRole := atlasv2.CloudRoleAssignment{
+		GroupId: &value[0], 
+		// TODO fix this
+		RoleName:    pointer.Get(strings.ToUpper(value[1])),
```
Comment (wtrocki) [inline internal/cli/atlas/users/invite.go:158]: Notice that this is renamed from `RoleName` from `Role`. 
Would that be correct?

**Comment by wtrocki** [comment]
Working on failed tests

File: internal/cli/atlas/cloudproviders/accessroles/list.go [hunk 3]
```diff
@@ -42,7 +42,7 @@ func (opts *ListOpts) initStore(ctx context.Context) func() error {
 }
 
 var listTemplate = `ID	PROVIDER	ATLAS AWS ACCOUNT ARN	UNIQUE EXTERNAL ID{{range .AwsIamRoles}}
-{{.RoleId}}	{{.ProviderName}}	{{.AtlasAWSAccountArn}}	{{.AtlasAssumedRoleExternalId}}{{end}}
+{{.RoleID}}	{{.ProviderName}}	{{.AtlasAWSAccountArn}}	{{.AtlasAssumedRoleExternalId}}{{end}}
```
Comment (matt-condon) [inline internal/cli/atlas/cloudproviders/accessroles/list.go:45]: Was this intentional?

**Review by matt-condon — APPROVED** [review]

**Comment by wtrocki** [comment]
@fmenezes Do we have planned releases for this week? I would love to merge this and work with follow up PRs that will be small and improve our test cases.
Is that on?

File: go.mod [hunk 4]
```diff
@@ -40,7 +40,7 @@ require (
 	github.com/tangzero/inflector v1.0.0
 	github.com/withfig/autocomplete-tools/packages/cobra v1.2.0
 	go.mongodb.org/atlas v0.27.0
-	go.mongodb.org/atlas-sdk v0.12.0
+	go.mongodb.org/atlas-sdk v1.0.0-rc2
```
Comment (wtrocki) [inline go.mod:43]: Will merge that version to test dependabot update

**Pull request merged**